arch/risc-v/Kconfig

```diff
@@ -147,6 +147,25 @@ config ARCH_MMU_TYPE_SV39
 	bool
 	default n
 
+# MPU has certain architecture dependent configurations, which are presented
+# here. Default is that the full RISC-V PMP specification is supported.
+
+config ARCH_MPU_MIN_BLOCK_SIZE
```

Comment:
These default values I'm not very happy with. I would like to configure this so that the default values are like the comment says; every configuration is supported and the minimum PMP block size is 4 bytes. When ARCH_CHIP_ MPFS is set, the values would change to these.

However, my Kconfig skills are a bit limited and I did not figure out a way to do this. Now the build process gives .config:107:warning: symbol value '' invalid for ARCH_MPU_MIN_BLOCK_SIZE for the other targets.